Using Git to retrieve the dropboxingore. (#7)
sp1thas/dropboxignore: README.md

### Basic Installation

> dropboxignore is installed by running one of the following commands in your terminal. You can install this via the command-line with either curl, wget or another similar tool. `attr` package should be installed on your system. **Git** is required to run the installation.  

Reviewer: ```suggestion
dropboxignore is installed by running one of the following commands in your terminal. You can install this via the command-line with either curl, wget or another similar tool. `attr` and `git` package should be installed on your system.
```
